back-end/src/main/java/com/skynet/skymed/controller/PessoaController.java

```diff
@@ -25,59 +27,76 @@
 public class PessoaController {
 
 	private EmailDePacienteService servicoDeEmailPaciente = new EmailDePacienteService();
+	private final String PACIENTE_INEXISTENTE = "Paciente inexistente.";
 
 	@Autowired
 	private PessoaRepository pessoaDB;
+	
+	@ExceptionHandler({ HttpMessageNotReadableException.class })
+    public ResponseEntity<Object> handleException(HttpMessageNotReadableException ex) {
+		return ResponseEntity.badRequest().body(ex.getMostSpecificCause().getMessage());
+    }
 
 	@GetMapping
-	public ResponseEntity<ArrayList<Pessoa>> getObject() {
+	public ResponseEntity<Object> getObject() {
 		var pessoas = pessoaDB.findAll();
-		if (pessoas.size() == 0) {
-			return ResponseEntity.status(HttpStatus.NOT_FOUND).body(null);
 
+		if (pessoas.size() == 0) {
+			return ResponseEntity.status(HttpStatus.NOT_FOUND).body("Nenhuma pessoa encontrada.");
 		}
 		return ResponseEntity.ok((ArrayList<Pessoa>) pessoas);
```

Comment:
```suggestion
		return ResponseEntity.ok(pessoas);
```
Pode mandar só pessoas, não precisa do cast